src/audio/multiband_drc/multiband_drc.h

```diff
@@ -21,6 +21,12 @@
 /**
  * Stores the state of the sub-components in Multiband DRC
  */
+struct multiband_drc_coefficients {
+	struct sof_eq_iir_biquad crossover[SOF_MULTIBAND_DRC_MAX_BANDS * PLATFORM_MAX_CHANNELS];
+	struct sof_eq_iir_biquad emphasis[SOF_MULTIBAND_DRC_MAX_BANDS * PLATFORM_MAX_CHANNELS];
+	struct sof_eq_iir_biquad deemphasis[SOF_MULTIBAND_DRC_MAX_BANDS * PLATFORM_MAX_CHANNELS];
+};
```

Comment:
the more I read this code the more I think this change is incorrect. Look at the struct directly below. This is a copy of that struct which is already directly embedded in the component data. How does this benefit anything when the data was already allocated in place in a single block?

Reply:
Thanks ! Please take a look at the latest changes.